### PR TITLE
9 DMU + some cleanup

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/bite_down.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Name:Bite Down
+ManaCost:1 G
+Types:Instant
+A:SP$ Pump | Cost$ 1 G | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ SoulsDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature or planeswalker you don't control.
+SVar:SoulsDamage:DB$ DealDamage | AILogic$ PowerDmg | ValidTgts$ Creature.OppCtrl,Planeswalker.OppCtrl | TgtPrompt$ Select creature or planeswalker you don't control.  | NumDmg$ X | DamageSource$ ParentTarget | SpellDescription$ This creature deals damage equal to its power to any target.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/bite_down.txt:5]: SpellDescription$ should match the oracle "to target creature or planeswalker you don't control."

Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/bite_down.txt:5]: TgtPrompt$ Select target creature

File: forge-gui/res/cardsfolder/upcoming/strength_of_the_coalition.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:Strength of the Coalition
+ManaCost:G
+Types:Instant
+K:Kicked:2 W
+A:SP$ Pump | ValidTgts$ Creature.youCtrl | NumAtt$ +2 | NumDef$ +2 | SubAbility$ DBCounter | SpellDescription$ Target creature you control gets +2/+2 until end of turn. If this spell was kicked, put a +1/+1 counter on each creature you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/strength_of_the_coalition.txt:5]: ValidTgts typo

File: forge-gui/res/cardsfolder/upcoming/tail_swipe.txt [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Name:Tail Swipe
+ManaCost:G
+Types:Instant
+A:SP$ Pump | ConditionPlayerTurn$ True | ConditionPhases$ Main1,Main2 | ConditionDefined$ Self | ConditionPresent$ Card.wasCast | NumAtt$ +1 | NumDef$ +1 | AILogic$ Fight | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Choose target creature you control | CounterType$ P1P1 | SubAbility$ DBFight | SpellDescription$ Choose target creature you control and target creature you don’t control. If you cast this spell during your main phase, the creature you control gets +1/+1 until end of turn. Then those creatures fight each other. (Each deals damage equal to its power to the other.)
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/tail_swipe.txt:4]: don’t has a curly

File: forge-gui/res/cardsfolder/upcoming/tail_swipe.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+SVar:DBFight:DB$ Fight | Defined$ ParentTarget | ValidTgts$ Creature.YouDontCtrl | TgtPrompt$ Choose target creature you don't control | StackDescription$ Then the chosen creatures fight each other. (They each deal damage equal to their power to the other.)
+DeckHas:Ability$Counters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tail_swipe.txt:6]: Wrong DeckHas

File: forge-gui/res/cardsfolder/upcoming/tail_swipe.txt (hunk 3 above)
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/tail_swipe.txt:4]: CounterType?

File: forge-gui/res/cardsfolder/upcoming/bite_down.txt [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Name:Bite Down
+ManaCost:1 G
+Types:Instant
+A:SP$ Pump | Cost$ 1 G | ValidTgts$ Creature.YouCtrl | AILogic$ PowerDmg | TgtPrompt$ Select target creature you control | SubAbility$ SoulsDamage | StackDescription$ None | SpellDescription$ Target creature you control deals damage equal to its power to target creature or planeswalker you don't control.
+SVar:SoulsDamage:DB$ DealDamage | AILogic$ PowerDmg | ValidTgts$ Creature.OppCtrl,Planeswalker.OppCtrl | TgtPrompt$ Select creature or planeswalker you don't control | NumDmg$ X | DamageSource$ ParentTarget | SpellDescription$ This creature deals damage equal to its power to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bite_down.txt:5]: Use `YouDontCtrl`

File: forge-gui/res/cardsfolder/upcoming/bite_down.txt (hunk 1 above)
Comment (paulsnoops) [reply forge-gui/res/cardsfolder/upcoming/bite_down.txt]: This doesn't look resolved, it shouldn't say any target?

Comment (paulsnoops) [reply forge-gui/res/cardsfolder/upcoming/bite_down.txt]: This doesn't look resolved, missing the word target I think in TgtPrompt

File: forge-gui/res/cardsfolder/upcoming/stall_for_time.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Stall for Time
+ManaCost:2 W
+Types:Instant
+K:Kicker:1 U
+A:SP$ Tap | TgtPrompt$ Select up to two target creatures | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 2 | SubAbility$ DBDraw | SpellDescription$ Tap up to two target creatures. If this spell was kicked, put a stun counter on each of those creatures. (If a permanent with a stun counter on it would become untapped, remove one from it instead.)
+SVar:DBDraw:DB$ Draw  | SubAbility$ DBCounter | SpellDescription$ Draw a card.
+SVar:DBCounter:DB$ PutCounter | Condition$ Kicked | Defined$ Targeted | CounterType$ Stun | CounterNum$ 3
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/stall_for_time.txt:7]: CounterNum$ 3 is wrong

File: forge-gui/res/cardsfolder/upcoming/bite_down.txt (hunk 1 above)
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/bite_down.txt]: Got it ! I think the spelldescription is just not needed here

File: forge-gui/res/cardsfolder/upcoming/stall_for_time.txt [hunk 7]
```diff
@@ -0,0 +1,10 @@
+Name:Stall for Time
+ManaCost:2 W
+Types:Instant
+K:Kicker:1 U
+A:SP$ Tap | TgtPrompt$ Select up to two target creatures | ValidTgts$ Creature | TargetMin$ 0 | TargetMax$ 2 | SubAbility$ DBDraw | SpellDescription$ Tap up to two target creatures. If this spell was kicked, put a stun counter on each of those creatures. (If a permanent with a stun counter on it would become untapped, remove one from it instead.)
+SVar:DBDraw:DB$ Draw  | SubAbility$ DBCounter | SpellDescription$ Draw a card.
```
Comment (paulsnoops) [inline forge-gui/res/cardsfolder/upcoming/stall_for_time.txt:6]: Also, should the draw come after the PutCounter here?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/stall_for_time.txt:6]: Draw needs to be after the Counter

File: forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt [hunk 8]
```diff
@@ -0,0 +1,12 @@
+Name:Stenn, Paranoid Partisan
+ManaCost:W U
+Types:Legendary Creature Human Wizard
+PT:2/2
+K:ETBReplacement:Other:ChooseCT
+SVar:ChooseCT:DB$ ChooseType | Defined$ You | Type$ Card | ValidTypes$ Artifact,Planeswalker,Enchantment,Instant,Sorcery | SpellDescription$ As CARDNAME enters the battlefield, choose a card type other than creature or land.
+S:Mode$ ReduceCost | ValidCard$ Card.ChosenType | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Spells you cast of the chosen type cost {1} less to cast.
+A:AB$ ChangeZone | Cost$ 1 W U | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig | SpellDescription$ Exile NICKNAME, Return it to the battlefield under its owner's control at the beginning of the next end step.
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | TriggerDescription$ Return CARDNAME to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | Defined$ Self | Origin$ Exile | Destination$ Battlefield
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt:10]: @Northmoc @tool4ever my gut is telling me that the delayed trigger probably needs to remember it

can you test what happens when it is removed from exile and exiled again?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt:10]: yea, otherwise the Self will be battlefield object with old timestamp and fail

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt:10]: i thought we were touching delayed trigger again so the host of the trigger isn't lki, so this still works what you don't need to extra remember it?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt:10]: Stuff like _Fog Elemental_ won't need it but here the zone change happens before the trigger is created though

File: forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt [hunk 9]
```diff
@@ -5,8 +5,9 @@ PT:2/2
...
 S:Mode$ ReduceCost | ValidCard$ Card.ChosenType | Type$ Spell | Activator$ You | Amount$ 1 | Description$ Spells you cast of the chosen type cost {1} less to cast.
-A:AB$ ChangeZone | Cost$ 1 W U | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig | SpellDescription$ Exile NICKNAME, Return it to the battlefield under its owner's control at the beginning of the next end step.
-SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | Phase$ End of Turn | Execute$ TrigReturn | TriggerDescription$ Return CARDNAME to the battlefield.
-SVar:TrigReturn:DB$ ChangeZone | Defined$ Self | Origin$ Exile | Destination$ Battlefield
+A:AB$ ChangeZone | Cost$ 1 W U | Origin$ Battlefield | Destination$ Exile | SubAbility$ DelTrig | RememberChanged$ True | SpellDescription$ Exile NICKNAME, Return it to the battlefield under its owner's control at the beginning of the next end step.
+SVar:DelTrig:DB$ DelayedTrigger | Mode$ Phase | RememberObjects$ Remembered | Phase$ End of Turn | Execute$ TrigReturn | TriggerDescription$ Return CARDNAME to the battlefield.
+SVar:TrigReturn:DB$ ChangeZone | Defined$ Remembered | Origin$ Exile | Destination$ Battlefield | SubAbility$ DBCleanup
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/stenn_paranoid_partisan.txt:10]: if you are going this route with DelayedTrigger, then the Cleanup needs to be a SubAbility of it

the ChangeZone needs to use `DelayTriggerRememberedLKI`

**Review by paulsnoops — APPROVED** [review]

**Pull request merged**